assets/stylesheets/main.css

```diff
@@ -0,0 +1,173 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 2. v2.0 | 20110126
+  License: none (public domain)
+*/
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/*
+  ========================================
+  Grid
+  ========================================
+*/
+
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+/*
+=============================================
+main-mobile css
+=============================================
+*/
+
+body {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+}
+
+.welcome {
+  font-weight: 600;
+  font-size: 20px;
+  padding-top: 54px;
+  padding-left: 6.4%;
+  color: #28352f;
+}
+
+.fa-bars {
+  position: absolute;
+  top: 57px;
+  right: 9.34%;
+}
+
+.header-mobile {
```

Comment:
- [ ] Kindly use flexbox to layout the elements in the header section: logo(left) and the hamburger button(right). This gives us a lot of options to align all our elements, plus, it's responsive.